tests/integration-tests/tests/update/test_update.py

```diff
@@ -190,6 +192,42 @@ def test_update_slurm(region, pcluster_config_reader, s3_bucket_factory, cluster
     # - Test extra json
 
 
+@pytest.mark.dimensions("us-west-1", "c5.xlarge", "*", "slurm")
+@pytest.mark.usefixtures("instance", "scheduler")
+@pytest.mark.parametrize(
+    "imds_secured, imds_secured_updated, users_allow_list",
+    [
+        (True, False, {"root": True, "pcluster": True, "slurm": True}),
+        (False, True, {"root": True, "pcluster": True, "slurm": False}),
+    ],
+)
+def test_update_imds_secured(
```

Comment:
* rather than adding a dedicated test for the update of this one parameter, would it be possible to instead 
* if adding a test is necessary, is it important to test the update in both directions (i.e., secured to not secured and vice versa)?

Reply:
1. Truncated sentence in your comment. Do you mean to add it to an existent test?
2. I think it is necessary to both test from secured, to unsecured and vice versa because different branch in the cookbook are executed in the two cases.

Reply:
ugh, my bad. meant to say: 'rather than adding a dedicated test for the update of this one parameter, would it be possible to instead add it to existing update tests?'

Reply:
I got your point and I think you're right in terms of frugality.
Nevertheless, even if it is possible, I don't think it would be a good approach to pack everything within a unique update test because we would end up having one giant test for everything.